### ISSUE TITLE
Piping into cvslens does not work on Mac (Monterey/Ventura/Intel/M1)

### ISSUE THREAD

**Issue by ondohotola** [YS-L/csvlens#20]
Piping (any CSV file) into `csvlens` on the Mac (Monterey, rust 1.65.0) bombs with this backtrace

```
thread 'main' panicked at 'not yet implemented', /Users/el/.cargo/registry/src/github.com-1ecc6299db9ec823/csvlens-0.1.10/src/util/events.rs:58:23
stack backtrace:
   0:        0x1021becb2 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::h9515a3bdbd0baca0
   1:        0x1021d253a - core::fmt::write::hfb97b04d44bdb4d8
   2:        0x1021a4c8c - std::io::Write::write_fmt::h686d9784d8415cb9
   3:        0x1021a763f - std::panicking::default_hook::{{closure}}::ha298777f08731ac8
   4:        0x1021a72cf - std::panicking::default_hook::hb5f65353eb4f3d66
   5:        0x1021a7d52 - std::panicking::rust_panic_with_hook::he911070515b336d8
   6:        0x1021befc2 - std::panicking::begin_panic_handler::{{closure}}::h752456daaf48f28e
   7:        0x1021bee08 - std::sys_common::backtrace::__rust_end_short_backtrace::hc10b12502582d9b6
   8:        0x1021a786d - _rust_begin_unwind
   9:        0x1021e3b93 - core::panicking::panic_fmt::hc606b9962be5fb74
  10:        0x1021e3a77 - core::panicking::panic::h5459910156e8485f
  11:        0x102087a18 - csvlens::util::events::CsvlensEvents::next::h4ebcf28b757c6d70
  12:        0x1020a6446 - csvlens::input::InputHandler::next::h48c5a29e4a233be3
  13:        0x102090e0d - csvlens::app::App::main_loop::h938a72257a5b9420
  14:        0x10208b44c - csvlens::main::h1977ea503b48407c
  15:        0x1020a0026 - std::sys_common::backtrace::__rust_begin_short_backtrace::ha497255ce33b7923
  16:        0x1020a08dc - std::rt::lang_start::{{closure}}::h77a3420c0367f8ce
  17:        0x1021a4338 - std::rt::lang_start_internal::h7de129d052efa930
  18:        0x10208baaf - _main
```

_Originally posted by @ondohotola in https://github.com/YS-L/csvlens/issues/12#issuecomment-1338940432_

**Comment by YS-L** [comment]
Thanks for reporting! I can reproduce this.

This is related to https://github.com/crossterm-rs/crossterm/issues/500. There is an on-going effort to remove this limitation in crossterm, and csvlens can pick up the fix once it landed upstream.

**Comment by YS-L** [comment]
This should be fixed now in v0.1.11.

**Comment by ondohotola** [comment]
I can confirm that this works. Thank you.
